src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/_Imports.razor

```diff
@@ -4,7 +4,6 @@
 @using Microsoft.AspNetCore.Components.Forms
 @using Microsoft.AspNetCore.Components.Routing
 @using Microsoft.AspNetCore.Components.Web
-@using Microsoft.AspNetCore.Components.Web.Virtualization
```

Comment:
Why is this removed?

Reply:
Why was this added in the first place? FYI, I discussed this a little more in https://github.com/dotnet/aspnetcore/pull/25456#issuecomment-684599620.

Reply:
Virtualization isn't wasm specific, it's part of the core component experience: https://github.com/dotnet/aspnetcore/blob/master/src/Components/Web/src/Virtualization/Virtualize.cs#L12-L18. 

It's weird that you're seeing the error. Perhaps you are using an an old targeting pack?

Reply:
Yeah, I believe the template is fine here and this is some environment-related issue. I see this PR is already merged but we should consider bringing it back in another PR. Perhaps @MackinnonBuck can resolve this in his virtualization improvements PR?

Reply:
I'm making a followup PR to this one to rc2 so I'll do it there. Can one of you help me figure out why I was running into the error during verification?